src/nuclio/functions/version/version-triggers/version-triggers.component.js

```diff
@@ -11,15 +11,17 @@
         });
 
     function NclVersionTriggersController($rootScope, $timeout, $i18next, i18next, lodash, DialogsService,
-                                          VersionHelperService) {
+                                          FunctionsService, VersionHelperService) {
         var ctrl = this;
         var lng = i18next.language;
+        var uniqueClasses = ['http'];
```

Comment:
👍 👍 👍 
Very nice!
This way if in the future we need to apply the same functionality to more trigger kinds, all it would take is to just add strings to this array.
That’s a very good example of how to write maintainable code.

Reply:
Thanks!